src/core/jpg.js

```diff
@@ -594,6 +605,39 @@ var JpegImage = (function JpegImageClosure() {
     return a <= 0 ? 0 : a >= 255 ? 255 : a;
   }
 
+  function findNextFileMarker(data, currentPos, startPos) {
+    function peekUint16(pos) {
```

Comment:
As agreed on IRC, since already is such a function (see https://github.com/mozilla/pdf.js/blob/cd5acf501e9899f386bb7efc9feded7b2c6e763b/src/core/jpg.js#L600), I'm leaving it as-is.